gtsam/basis/Chebyshev.cpp

```diff
@@ -0,0 +1,97 @@
+/* ----------------------------------------------------------------------------
+
+ * GTSAM Copyright 2010, Georgia Tech Research Corporation,
+ * Atlanta, Georgia 30332-0415
+ * All Rights Reserved
+ * Authors: Frank Dellaert, et al. (see THANKS for the full author list)
+
+ * See LICENSE for the license information
+
+ * -------------------------------------------------------------------------- */
+
+/**
+ * @file Chebyshev.cpp
+ * @brief Chebyshev basis decompositions
+ * @author Varun Agrawal, Frank Dellaert
```

Comment:
Add Jing Dong to all code copied from DoubleExpresso

Reply:
Doing for all code except `BasisFactors.h` since I explicitly wrote that (when we switched away from Expressions).